SpatialGDK/Source/SpatialGDKServices/Public/SpatialCommandUtils.h

```diff
@@ -37,8 +37,12 @@ class SpatialCommandUtils
 	SPATIALGDKSERVICES_API static bool FetchPackageBinary(const FString& PackageVersion, const FString& PackageExe,
 														  const FString& PackageName, const FString& SaveLocation,
 														  const bool bIsRunningInChina, const bool bUnzip);
+	SPATIALGDKSERVICES_API static bool FetchPackageBinaryWithRetries(const FString& PackageVersion, const FString& PackageExe,
+																	 const FString& PackageName, const FString& SaveLocation,
+																	 const bool bIsRunningInChina, const bool bUnzip,
+																	 const int32 NumRetries = 3);
 
 private:
 	// Timeout given in seconds.
-	static constexpr double ProcessTimeoutTime = 60.0;
+	static constexpr double ProcessTimeoutTime = 120.0;
```

Comment:
Wonder if we should make this a setting somehow, this was my intention when first making this timeout but it slipped by me. Your discretion